public/src/js/main.js

```diff
@@ -0,0 +1,102 @@
+// This function will be called in every html page other than index due to path's being different
+function populateNavbar(){
+    let headTag = document.querySelector('#navbar');
+    headTag.innerHTML = `
+    <div class="navbar-brand">
+        <a class="navbar-item" href="../../index.html">
```

Comment:
This is fine for starters, but you loose IDE intellisense with having HTML in a string like that. In [one of my projects](https://github.com/ANDREYDEN/ANDREYDEN.github.io/blob/master/js/main.js#L66) I have used `fetch` to load some html dynamically. For example, you could do something like 
```
fetch('src/html/profile.html')
    .then(data => data.text()
        .then(text => console.log(text)
    )
)
```
to get the profile html